timelock-agent/src/main/java/com/palantir/timelock/config/ClusterConfiguration.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ */
+
+package com.palantir.timelock.config;
+
+import org.immutables.value.Value;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.google.common.base.Preconditions;
+import com.palantir.remoting2.config.service.ServiceConfiguration;
+
+@JsonDeserialize(as = ImmutableClusterConfiguration.class)
+@JsonSerialize(as = ImmutableClusterConfiguration.class)
+@Value.Immutable
+public interface ClusterConfiguration {
+    // in remoting3 this class is officially public API
+    // TODO: do we want to live reload this? probably not? how do we make a timelock cluster bigger?
+    ServiceConfiguration cluster();
```

Comment:
Thoughts on a better name for this or the other? At the moment we have nested `cluster` objects in the config:
```
install: # TimeLockInstallConfiguration
  cluster: # ClusterConfiguration
    cluster: # ServiceConfiguration
    local-server: # String
```